modules/auxiliary/scanner/smb/smb_version.rb

```diff
@@ -200,6 +200,9 @@ def run_host(ip)
 
         info = smb_proto_info
         desc = "SMB Detected (versions:#{info[:versions].join(', ')}) (preferred dialect:#{info[:preferred_dialect]})"
+        if info[:versions].include?(1)
+          desc << ' (SMBv1: true)'
+        end
```

Comment:
Wouldn't this be covered on the previous line where `(version:` includes 1? This seems redundant. I would expect a server that supports SMBv1 to show `(versions:1)` or `(versions:1,2)`, etc.